demo/nextjs/components/sign-up.tsx

```diff
@@ -151,26 +150,25 @@ export function SignUp() {
 						className="w-full"
 						disabled={loading}
 						onClick={async () => {
-							await signUp.email({
-								email,
-								password,
-								name: `${firstName} ${lastName}`,
-								image: image ? await convertImageToBase64(image) : "",
-								callbackURL: "/dashboard",
```

Comment:
I removed the `callbackURL` here, since I found that onSuccess will not be called if it exists. Not sure if this is a bug, but we should write this behavior in the document